tests/LtiDeepLinkResourceTest.php

```diff
@@ -139,11 +172,26 @@ public function testItSetsTarget()
 
     public function testItCastsToArray()
     {
+        $icon = (new LtiDeepLinkResourceIcon())
```

Comment:
Completely optional suggestion, but it looks cleaner when chained IMO:
```suggestion
        $icon = LtiDeepLinkResourceIcon::new()
```